### ISSUE TITLE
wrapArguments unexpectedly inserting spaces in 0.47

### ISSUE THREAD

**Issue by calda** [nicklockwood/SwiftFormat#773]
In `0.47`, the `wrapArguments` rule is unexpectedly inserting spaces on blank lines that are the first line in an array literal:

`swiftformat . --rules wrapArguments --wrapcollections before-first`

```diff
 // wrapArguments inserts two spaces on the blank like before "Foo"

 let arrayWithBlankLines: [
-
+   
   "Foo",
   "Bar"

 ]
```

This seems unexpected, and didn't happen in `0.46`. This is causing conflicts with SwiftLint's `trailing_whitespace` rule (since SwiftFormat is unexpectedly inserting trailing whitespace).

**Comment by nicklockwood** [comment]
@calda this looks like a bug, but it can probably be fixed by enabling SwiftFormat's `trailingSpace` rule.

**Comment by calda** [comment]
aah we should use the SwiftFormat rule instead of the SwiftLint rule here (the SwiftLint rule doesn't autocorrect)

**Comment by calda** [comment]
I retried this with `trailingSpace` enabled as well and it still inserted the spaces

(`swiftformat . --rules wrapArguments,trailingSpace --wrapcollections before-first`)

**Comment by nicklockwood** [comment]
@calda this should now be fixed on develop

**Comment by nicklockwood** [comment]
(trailingSpace was being run before wrapArguments, which is why it didn't help with the issue)

**Comment by calda** [comment]
cool, thanks!

**Comment by nicklockwood** [comment]
Fixed in 0.47.1
